src/screens/AnimationScreen.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import { View, Image, StyleSheet, Animated, Easing } from 'react-native';
+
+export const ANIMATION_SCENE_NAME = 'ANIMATION_SCENE';
+
+export default class AnimationScreen extends Component {
+
+    state = {
+        topX: new Animated.Value(0),
+      }
+
+  static navigationOptions = {
+    title: 'Animation',
+  };
+
+  componentDidMount() {
+    this.goDown(); 
+  }
+
+  goDown(){
+    Animated.timing(                
+        this.state.topX,           
+        {
+          toValue: 200,
+          easing: Easing.ease,            
+          duration: 1000             
+        }
+      ).start(() => this.goTop());
+  }
+
+  goTop(){
+    Animated.timing(                
+        this.state.topX,           
+        {
+          toValue: 0,
+          easing: Easing.ease,            
+          duration: 1000             
+        }
+      ).start(() => this.goDown());
+  }
+  
+
+  render() {
+    let { topX } = this.state;
```

Comment:
On pourrait directement, au lieu de créer une variable qui ne sert que pour un cas, mettre directement this.state.topX dans le style de L'Animated.View